lib/transactions.js

```diff
@@ -83,12 +93,60 @@ export const renderDetailsString = ({
     intl.locale,
   );
 
-  const detailString = [amountString, hostFeeString, platformFeeString, paymentProcessorFeeString];
-  if (taxAmount) {
-    detailString.push(formatFee(taxAmount, amount, taxInfo?.id || 'Taxes', taxInfo?.percentage, intl.locale));
-  }
+  if (kind === TransactionKind.EXPENSE) {
+    const expensePaymentProcessorFee = formatCurrency(
+      Math.abs(paymentProcessorFee.valueInCents),
+      paymentProcessorFee.currency,
+      { locale: intl.locale },
+    );
+    const netExpenseAmount = formatCurrency(Math.abs(_netAmount.valueInCents), amount.currency, {
+      locale: intl.locale,
+    });
+    const payee = toAccount.name;
+    const payer = fromAccount.name;
+    const feesPayer = expense.feesPayer;
 
-  return detailString.concat(' ');
+    return (
+      <React.Fragment>
+        <Container mb={1}>
+          {paymentProcessorFee.valueInCents !== 0 ? (
+            <FormattedMessage defaultMessage="Expense Amount: {expenseAmount}" values={{ expenseAmount }} />
+          ) : (
+            netExpenseAmount
+          )}
+        </Container>
+        {paymentProcessorFee.valueInCents !== 0 && (
```

Comment:
I don't understand what we're displaying if `paymentProcessorFee.valueInCents === 0`. How does that work?

Reply:
We are displaying as per my previous comment; https://github.com/opencollective/opencollective-frontend/pull/7391#discussion_r804875148; although this will change slightly with my PR for the other issue; https://github.com/opencollective/opencollective/issues/5220 to show the refunded payment processor fee.